src/lib/optimizer/abstract_syntax_tree/aggregate_node.cpp

```diff
@@ -78,24 +78,24 @@ std::string AggregateNode::description() const {
 std::string AggregateNode::get_verbose_column_name(ColumnID column_id) const {
```

Comment:
The changes here fix a bug when printing AggregateNodes (wrong order of columns assumed).